aircontrol

```diff
@@ -7,12 +7,27 @@ else
 fi
 
 read -d '' APPLESCRIPT <<EOF
-tell application "System Events"
+set _isElCapitanOrBetter to (system attribute "sysv") ≥ 4245
```

Comment:
`≥ 4245` didn't work for me since that's the current version for macOS Yosemite. `> 4245` should work but there could be an issue later if `10.10.6` is released.

How about this instead:
```
set _isElCapitanOrBetter to (system attribute "sys1") ≥ 10 and (system attribute "sys2") ≥ 11
```

Admittedly, `(system attribute "sys1") ≥ 10` is superfluous here 😄 but that can't hurt either. Using `sys1` and `sys2` has the advantage of being explicit.